xbmc/listproviders/DirectoryProvider.cpp

```diff
@@ -535,7 +547,18 @@ class CVideoPlayActionProcessor : public VIDEO::GUILIB::CVideoPlayActionProcesso
 
   bool OnPlaySelected() override
   {
-    ExecuteAction({"PlayMedia", m_item, "noresume"});
+    if (m_item.GetVideoInfoTag()->m_type == MediaTypeMovie)
```

Comment:
Could you please add a `//! @todo get rid of special handling for movie versions`, so we can pick this up in a follow-up PR?